src/tests/api/claimedIdentity.test.ts

```diff
@@ -0,0 +1,75 @@
+import { assert200OK, assert400BadRequest, assert401Unauthorized } from "../../utils/apiCommonAssertions";
+import { claimedIdentityPost, sessionPost, stubStartPost } from "../../utils/apiTestSteps";
+
+describe("Happy Path Post Claimed Identity", () => {
+    
+	let sessionResponse: any;
+
+	beforeEach( async () =>{
+		const requestBody = {
+			target:"https://unb2uhs7rl.execute-api.eu-west-2.amazonaws.com/dev/",
+		};
+		const stubResponse = await stubStartPost(requestBody);
+
+		sessionResponse = await sessionPost(stubResponse.data.clientId, stubResponse.data.request);
+
+	});
+    
+	it("Given claimed identity has been posted", async () => {
+
+		const sessionId = sessionResponse.data.session_id;
+  
+		const response = await claimedIdentityPost(
+			"Slim", "Pickens", "1970-03-26", "brp", "2024-03-31", sessionId,
+		);
+ 
+		assert200OK({ status: response.status, statusText: response.statusText });
+
+	});
+
+});
+
+
+// describe("UnHappy Path", () => {
```

Comment:
Are we a 'remove commented out code' repo @richa-misra?

Reply:
Commented out code has been removed